latch/types/metadata.py

```diff
@@ -101,10 +166,14 @@ class LatchParameter:
     batch_table_column: bool = False
     appearance_type: LatchAppearanceType = LatchAppearanceType.line
     rules: List[LatchRule] = field(default_factory=list)
+    detail: Optional[str] = None
+    _custom_ingestion: Optional[str] = None
 
     def __str__(self):
-        metadata_yaml = yaml.dump(yaml.safe_load(json.dumps(self.dict)))
-        return f"{self.description}\n{metadata_yaml}"
+        metadata_yaml = yaml.safe_dump(self.dict, sort_keys=False)
```

Comment:
why not sort keys

Reply:
made the resulting yaml easier to read during debugging, not necessary in prod i dont think